### DISCUSSION TITLE
【2023.12.15】275850 No Man's Sky

### DISCUSSION THREAD

**Discussion by HUMHIHello** [pjy612/SteamManifestCache#46] [Share]
appid 275850

[【2023.12.15】275850 No Man's Sky.zip](https://github.com/pjy612/SteamManifestCache/files/13686684/2023.12.15.275850.No.Man.s.Sky.zip)

**Comment by HUMHIHello** [comment]
appid:275850
keys
228983:77c8e812cd79e67e2d376721253ebb07e06b3646f05671c6c9517b27be14734b
228984:df7df98d3134f5e0009d5ebcaaf3bbb91ea2f7cbad81a37a9d16bc1835f57068
228985:21a2f639584d9bd462c5c4b3e10f881cba8a17ae674c830c4f083551cd356e9c
228988:1845444d5e2cfd0ae65ae4a8fedb6e2fbf776fcc5b913ab4ac461bc9a74f8358
228989:ad69276eb476cf06c40312df7376d63deac0c838b9a2767005be8bb306ffb853
228990:44d8c45ce229a11c4f231a3d2a350eaf80b0d69a8af938ec7ccca720f694b0e8
229007:eba9fa9ff4f811929c119074450f49f739113bcae9317ded9bea262aa21ec58d
229032:15b3308d4e95d1156182211bab1c823b6df0a06fd74a5c0833abed89bc307cfb
275851:a5ce5217e5d5d20115ea81209764d4e89f37bed55b84690951d5df23e73852a6

**Reply by pjy612** [reply]
如何联系？

**Reply by HUMHIHello** [reply]
hihello_0@163.com

**Comment by HUMHIHello** [comment]
我获得到的密匙与清单和steamdb上显示的不一致
不知道为什么
可能是这个游戏近期刚更新过一次吧
steamdb还没有更新?

**Reply by pjy612** [reply]
![image](https://github.com/pjy612/SteamManifestCache/assets/4072526/32e3fcee-f146-479e-8533-5aac62c72a9d)
一般只要这些就行。
[Shared Install] 的 一般都是免费能获取的

**Reply by HUMHIHello** [reply]
275852并没有获取 
可能是因为我系统版本问题

**Reply by HUMHIHello** [reply]
我又确认了一下 的确没有获取 
这是我修改后的密匙与清单 
keys:
275851:a5ce5217e5d5d20115ea81209764d4e89f37bed55b84690951d5df23e73852a6
[【2023.12.15】275850 No Man's Sky.zip](https://github.com/pjy612/SteamManifestCache/files/13687021/2023.12.15.275850.No.Man.s.Sky.zip)

**Reply by pjy612** [reply]
是的 他们 win 环境 有 275851 就够了
